apps/meteor/app/api/server/v1/users.ts

```diff
@@ -1000,6 +1000,10 @@ API.v1.addRoute(
 					throw new Meteor.Error('error-invalid-user-id', 'Invalid user id');
 				}
 
+				if (!settings.get('Accounts_TwoFactorAuthentication_Enabled')) {
+					throw new Meteor.Error('error-two-factor-not-enabled', 'Two factor authentication is not enabled');
+				}
+
```

Comment:
E2E can be used regardless of 2FA status. Blocking the reset E2E option when 2FA is disabled makes no sense.